sqlitedict.py

```diff
@@ -160,15 +160,13 @@ def __init__(self, filename=None, tablename='unnamed', flag='c',
                 raise RuntimeError('Error! The directory does not exist, %s' % dirname)
 
         self.filename = filename
-        if '"' in tablename:
-            raise ValueError('Invalid tablename %r' % tablename)
-        self.tablename = tablename
+        self.tablename = tablename.replace('"', '""')
```

Comment:
Add a comment here please, + link to this ticket (for future maintainers).

Reply:
Linked to the relevant part of the SQL 2002 standard.